src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_OF_BALL_COUNT = 100;
+    private static final Random RANDOM = new Random();
```

Comment:
you can make this constant `public` and then reuse it in `CollorSupplier` for getting a random color

this way you'll have only 1 instance of `Random`